raphtory/src/db/task/edge/window_eval_edge.rs

```diff
@@ -229,7 +228,8 @@ impl<'a, G: GraphViewOps, CS: ComputeState, S: 'static> EdgeViewOps
 
     fn history(&self) -> Vec<i64> {
         self.graph()
-            .edge_history_window(self.ev, self.t_start..self.t_end)
+            .edge_window_exploded(self.ev, self.t_start..self.t_end, self.g.layer_ids())
```

Comment:
Oh was this missing here? Need some more tests for the algos :D

Reply:
just a small refactor so we can completely delete a whole file :)